docs/source/user_guide/gcm_based_inference/answering_causal_questions/computing_counterfactuals.rst

```diff
@@ -35,7 +39,7 @@ Next, we'll model cause-effect relationships as an invertible SCM and fit it to
 
 >>> gcm.fit(causal_model, training_data)
 
-Finally, let's compute the counterfactual when intervening on X:
+Finally, let's compute the counterfactual when intervening on :math:`X`:
```

Comment:
I would replace "when intervening on ..." with "had we intervened on ...". Reads better. Clearly states we are contemplating about the past.

Reply:
This sentence changed in the new version.